_posts/2022-11-09-OpenSearch-Partner-Highlight-Using-Cloaked-Search.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "OpenSearch Partner Highlight: Using Cloaked Search to Protect Your Data"
```

Comment:
```suggestion
title:  "OpenSearch partner highlight: Using Cloaked Search to protect your data"
```

Reply:
Global: I have changed the title and headings throughout to sentence case, and I have changed mentions of "application-layer encryption" following the definition to "ALE".

Reply:
@natebower thanks for the careful review! I incorporated the changes in one commit instead of committing the individual suggestions.

Reply:
@BobWall23 My pleasure :-)